src/modules/logger/logger.h

```diff
@@ -58,6 +58,19 @@ namespace px4
 namespace logger
 {
 
+enum {
+	SDLOG_MODE_ARM_UNTIL_DISARM = 0,
```

Comment:
I prefer an `enum class` here for better namespacing: `enum class SDLogMode { ARM_UNTIL_DISARM...`